src/App.js

```diff
@@ -1,14 +1,20 @@
 import React, { Component } from "react";
 import AppRoutes from "./routes";
+import "./assets/css/App.scss";
+import "./assets/articleAssets/demo2.scss";
 import "./assets/authAssets/socialAuth.css";
 import { Provider } from "react-redux";
-import configureStore from "./store/";
-
+import store from "./store/";
+import { ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.min.css";
 class App extends Component {
```

Comment:
Always skip a line between imports and the component. Ensure to fix that in all files

Reply:
I have Added a new line for all files which a missing line after imports